.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "res/gear"]
```

Comment:
I'm not sure that git submodule is the optimal solution. I'd just downloaded zip archive from GitHub or make `git clone`.

Reply:
the other benifits are we can use the examples inside the gear repo also for testing, we can remove it and seperate the `builder` feature to another crate in the future also ( can not make big changes in the directory since there are some PRs in parallel  ), let's see